<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-R589HK7" failures="1" tests="10" name="test.SearchHotelTest" time="55.862" errors="0" timestamp="2025-04-08T13:12:03 IST" skipped="0">
  <testcase classname="test.SearchHotelTest" name="testLinksDispay" time="2.305"/>
  <testcase classname="test.SearchHotelTest" name="testSearchHotel" time="3.268"/>
  <testcase classname="test.SearchHotelTest" name="bookingWithPastDate" time="3.844">
    <failure message="if continue button is displayed fail this test case -&gt; expected [false] but found [true]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: if continue button is displayed fail this test case -> expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertFalse(Assert.java:79)
at test.SearchHotelTest.bookingWithPastDate(SearchHotelTest.java:56)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- bookingWithPastDate -->
  <testcase classname="test.SearchHotelTest" name="searchWithoutLocation" time="1.071"/>
  <testcase classname="test.SearchHotelTest" name="testSelectHotel" time="10.503"/>
  <testcase classname="test.SearchHotelTest" name="testSelectHotel" time="11.347"/>
  <testcase classname="test.SearchHotelTest" name="testSelectHotel" time="9.068"/>
  <testcase classname="test.SearchHotelTest" name="testSelectHotel" time="10.415"/>
  <testcase classname="test.SearchHotelTest" name="testWithoutSelectHotel" time="2.501"/>
  <testcase classname="test.SearchHotelTest" name="testWithoutOptionalFields" time="1.540"/>
</testsuite> <!-- test.SearchHotelTest -->
